api/v1alpha1/hostedcluster_types.go

```diff
@@ -149,8 +149,25 @@ type ClusterNetworking struct {
 	ServiceCIDR string `json:"serviceCIDR"`
 	PodCIDR     string `json:"podCIDR"`
 	MachineCIDR string `json:"machineCIDR"`
+	// NetworkType specifies the SDN provider used for cluster networking.
+	// +kubebuilder:default:="OpenShiftSDN"
+	// +unionDiscriminator
+	// +required
```

Comment:
required with default does not play well, client side validation will complain if not explicitly set. See https://github.com/openshift/hypershift/pull/169#issuecomment-838549331

Reply:
great catch! Thank you